assets/css/twenty-twenty-one.scss

```diff
@@ -2111,13 +2115,40 @@ a.reset_variations {
 		align-items: stretch;
 		flex-direction: row;
 		flex-wrap: wrap;
+		box-sizing: border-box;
+
+		&.columns-2 {
+			li.product {
+				width: 50% !important;
+			}
+		}
+
+		&.columns-3 {
+			li.product {
+				width: 33.33% !important;
+			}
+		}
+
+		&.columns-5 {
+			li.product {
+				width: 20% !important;
+			}
+		}
+
+		&.columns-6 {
+			li.product {
+				width:16.66% !important;
+			}
+		}
 
 		li.product {
 			display: flex;
 			flex-direction: column;
 			justify-content: space-between;
 			align-items: flex-start;
-			margin-bottom: 5em;
+			padding: 0 8px 16px 8px !important;
+			margin: 0;
```

Comment:
Calling out my own change here while I'm noticing it. I switched what was `margin` to `padding` so that the `border-box` calculations would be correct, but this appears to be affecting other pages.

Can probably switch this back to margin and account for it in the width of `.columns-n li.product` using `calc`.